raft/src/main/java/org/apache/kafka/raft/RaftConfig.java

```diff
@@ -93,8 +99,8 @@ public String toString() {
                 QUORUM_VOTERS_DOC)
             .define(QUORUM_ELECTION_TIMEOUT_MS_CONFIG,
                 ConfigDef.Type.INT,
-                30000,
-                atLeast(0),
+                5000,
+                atLeast(0L),
```

Comment:
nit: we can set the default value of jitter below to be the same as the election timeout for now.